src/client.js

```diff
@@ -65,6 +66,18 @@ function fetchResourcesForProducts(products, client) {
   }, []);
 }
 
+function normalizeAccessToken(config) {
+  if (config.accessToken) {
+    console.warn("[ShopifyBuy] accessToken is deprecated as of v1.0, please use storefrontAccessToken instead.")
+  }
+  const accessToken = config.storefrontAccessToken || config.accessToken;
+
+  return new Config({
+    domain: config.domain,
+    storefrontAccessToken: accessToken
+  });
```

Comment:
Also, this normalization should probably all be happening in the config constructor. (sorry for all the comments)